plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/pad.cpp

```diff
@@ -0,0 +1,161 @@
+    testing::ValuesIn(padMode),                                //
+    testing::ValuesIn(netPrecisions),                          //
+    testing::Values(InferenceEngine::Precision::UNSPECIFIED),  //
+    testing::Values(InferenceEngine::Precision::UNSPECIFIED),  //
+    testing::Values(InferenceEngine::Layout::ANY),             //
+    testing::Values(std::vector<size_t>{5}),                   //
+    testing::Values(CommonTestUtils::DEVICE_PLAIDML)           //
+);
+
+INSTANTIATE_TEST_CASE_P(           //
+    smoke_Pad1D,                   //
+    PadLayerTest,                  //
+    pad1Dparams,                   //
+    PadLayerTest::getTestCaseName  //
+);
+
+const std::vector<std::vector<int64_t>> padsBegin2D = {{0, 0}, {1, 1}, {2, 0}, {0, 3}};
+const std::vector<std::vector<int64_t>> padsEnd2D = {{0, 0}, {1, 1}, {0, 1}, {3, 2}};
+
+const auto pad2DConstparams = testing::Combine(                //
+    testing::ValuesIn(padsBegin2D),                            //
+    testing::ValuesIn(padsEnd2D),                              //
+    testing::ValuesIn(argPadValue),                            //
+    testing::Values(ngraph::helpers::PadMode::CONSTANT),       //
+    testing::ValuesIn(netPrecisions),                          //
+    testing::Values(InferenceEngine::Precision::UNSPECIFIED),  //
+    testing::Values(InferenceEngine::Precision::UNSPECIFIED),  //
+    testing::Values(InferenceEngine::Layout::ANY),             //
+    testing::Values(std::vector<size_t>{13, 5}),               //
+    testing::Values(CommonTestUtils::DEVICE_PLAIDML)           //
+);
+
+INSTANTIATE_TEST_CASE_P(           //
+    smoke_Pad2DConst,              //
+    PadLayerTest,                  //
+    pad2DConstparams,              //
+    PadLayerTest::getTestCaseName  //
+);
+
+const auto pad2Dparams = testing::Combine(                     //
+    testing::ValuesIn(padsBegin2D),                            //
+    testing::ValuesIn(padsEnd2D),                              //
+    testing::Values(0),                                        //
+    testing::ValuesIn(padMode),                                //
+    testing::ValuesIn(netPrecisions),                          //
+    testing::Values(InferenceEngine::Precision::UNSPECIFIED),  //
+    testing::Values(InferenceEngine::Precision::UNSPECIFIED),  //
+    testing::Values(InferenceEngine::Layout::ANY),             //
+    testing::Values(std::vector<size_t>{13, 5}),               //
+    testing::Values(CommonTestUtils::DEVICE_PLAIDML)           //
+);
+
+INSTANTIATE_TEST_CASE_P(           //
+    smoke_Pad2D,                   //
+    PadLayerTest,                  //
+    pad2Dparams,                   //
+    PadLayerTest::getTestCaseName  //
+);
+
+const std::vector<std::vector<int64_t>> padsBegin4D = {{0, 0, 0, 0}, {0, 3, 0, 0}, {0, 0, 0, 1},
+                                                       {0, 0, 1, 1}, {2, 0, 0, 0}, {0, 3, 0, 1}};
```

Comment:
Please add `,` after the last element so that clang format will format it as one element per line.